dotCMS/src/main/webapp/html/portlet/ext/workflows/workflows_js_inc.jsp

```diff
@@ -125,89 +125,73 @@
                     return;
 	    		}
 	    		else{
-	    			dojo.byId("wfActionId").value=this.wfActionId;
 
-	    			this.executeWorkflow();
+	    			this.executeWorkflow(wfId, inode);
 	    		}
 	    	},
 	    	
-	    	executeWorkflow : function (){
-	    		
-	    		dijit.byId('savingContentDialog').show();
-
-				var xhrArgs = {
-					form: dojo.byId("submitWorkflowTaskFrm"),
-					handleAs: "text",
-					load: function(data) {
-						showDotCMSSystemMessage("<%=LanguageUtil.get(pageContext, "Workflow-executed")%>");
-
-                        var customEvent = document.createEvent("CustomEvent");
-                        customEvent.initCustomEvent("ng-event", false, false,  {
-                            name: "edit-task-executed-workflow"
-                        });
-                        document.dispatchEvent(customEvent);
-					
-					},
-					error: function(error) {
-						showDotCMSSystemMessage(error);
-						dijit.byId('savingContentDialog').hide();
-					
-					}
-				}
-				dojo.xhrPost(xhrArgs);
-
+	    	executeWorkflow : function (wfId, inode){

```

Comment:
using the same approach as the integration of the PP modals, https://github.com/dotcms/core/issues/15074